src/Form/Form.js

```diff
@@ -45,31 +49,38 @@ class Form extends React.Component {
     await this.props.getQuestions(this.state.round6, 'hard')
     this.props.isLoading(false)
   }
-
+    
   renderInputs = () => {
     let inputs = []
     for(let i = 1; i < 7; i++) {
       inputs.push(
         <section className='round-label nes-select is-dark' key={i}>
-          <h3>{`Round ${i}`}</h3>
-          <select 
-            name={`round${i}`}
-            onChange={this.handleChange}
-            data-testid={`round${i}`}
+        <h3>{`Round ${i}`}</h3>
+        <select 
+          name={`round${i}`}
+          onChange={this.handleChange}
+          data-testid={`round${i}`}
           >
-            <option>Select a subject...</option>
-            <option value='23'>History</option>
-            <option value='22'>Geography</option>
-            <option value='19'>Math</option>
-            <option value='25'>Art</option>
-            <option value='17'>Science & Nature</option>
-            <option value='10'>Books</option>
-          </select>
-        </section>
-      )
-    }
-    return inputs
+          <option>Select a subject...</option>
+          <option value='23'>History</option>
+          <option value='22'>Geography</option>
+          <option value='19'>Math</option>
+          <option value='25'>Art</option>
+          <option value='17'>Science & Nature</option>
+          <option value='10'>Books</option>
+        </select>
+      </section>
+    )
   }
+  return inputs
+}
+  
+  // componentDidUpdate(prevProps, prevState) {
+  //   if (this.state !== prevState && this.state.name && this.state.round1 && this.state.round2 && this.state.round3 && this.state.round4 && this.state.round5 && this.state.round6) {
```

Comment:
Confused why this isn't working in a component did update.  My guess is that the new state object, despite having all of the same properties and values, is a different obj.  We can make this happen but it will take more line of code.